### PR TITLE
DOC-19: Added windows section for Sophos

### PR DESCRIPTION
**Pull request by nico-shishkin**
# What changed

Added Windows section - https://deploy-preview-1177--logz-docs.netlify.app/shipping/security-sources/sophos.html#windows

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/logzio_collections/_security-sources/sophos.md [hunk 1]
```diff
@@ -108,3 +118,122 @@ The logs will require customized parsing so they can be effectively mapped in Ki
...
+    - <<FILE_PATH>>
+  fields:
+    token: <<LOG-SHIPPING-TOKEN>>
+  fields_under_root: true
+  json.keys_under_root: true
+  encoding: utf-8
+  ignore_older: 3h
+
+#For version 7 and higher
+filebeat.registry.path: /var/lib/filebeat
```
Comment (boofinka) [inline _source/logzio_collections/_security-sources/sophos.md:176]: needs to be 'C:\ProgramData\Filebeat' and not /var/lib/filebeat

**Review by boofinka — CHANGES_REQUESTED** [review]
In line 176, the value of filebeat.registry.path: should be 'C:\ProgramData\Filebeat' and not /var/lib/filebeat

File: _source/logzio_collections/_security-sources/sophos.md (hunk 1 above)
Comment (nico-shishkin) [reply _source/logzio_collections/_security-sources/sophos.md]: Thanks Yotam, amended

Comment (boofinka) [reply _source/logzio_collections/_security-sources/sophos.md]: @nico-shishkin almost - the path should be inside single brackets.
`'C:\ProgramData\Filebeat'` Not `C:\ProgramData\Filebeat`

Comment (nico-shishkin) [reply _source/logzio_collections/_security-sources/sophos.md]: Thanks Yotam, resolved.

**Review by boofinka — APPROVED** [review]
Looks good

**Review by YajB — APPROVED** [review]

**Pull request merged**